apis/api/v1alpha1/conversion.go

```diff
@@ -18,7 +18,77 @@
 
 package v1alpha1
 
-// Hub marks this type as a conversion hub.
-func (*Astarte) Hub()               {}
-func (*AstarteVoyagerIngress) Hub() {}
-func (*Flow) Hub()                  {}
+import (
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+
+	"github.com/astarte-platform/astarte-kubernetes-operator/apis/api/v1alpha2"
+)
+
+// ConvertTo converts this Astarte to the Hub version (v1alpha2).
+func (src *Astarte) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1alpha2.Astarte)
+	err := Convert_v1alpha1_Astarte_To_v1alpha2_Astarte(src, dst, nil)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Could we shorten those lines like this? If there is no error, `nil` will be returned anyway.
```go
return Convert_v1alpha1_Astarte_To_v1alpha2_Astarte(src, dst, nil)
```
The same applies to all other conversion functions in this file.